script/infoRelease.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Absolute path to this script, e.g. /home/user/xxx/infoRlease.sh
-SCRIPT=$(readlink -f "$0")
+SCRIPT="$(cd $(dirname "$0") && pwd)/$(basename "$0")"
```

Comment:
The "readlink -f" command does not work in some versions of the OS/bash. AFAIK, the trick with the dirname/basename does. Result is the same.